packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -84,7 +89,7 @@ module.exports = {
   // Configure your compilers
   compilers: {
     solc: {
-      version: "0.8.15",      // Fetch exact version from solc-bin (default: truffle's version)
+      version: "0.8.15" // Fetch exact version from solc-bin (default: truffle's version)
```

Comment:
This indent should stay.

Reply:
Yeah! I forgot to add `--no-verify` tag in the commit to get pass the prettier. I will add that again. Thanks for catching this @cliffoo.

Reply:
Please don't use the `--no-verify` flag when committing, as that's the process.

Reply:
`--no-verify` flag is only used with `truffle-config.js` when committing due to the fact that `prettier` removes the trailing commas in that file, which is required for user convenience. There should be a way to exclude this file from using `prettier`, which I guess can be a separate PR to resolve this issue.